lib/mayaUsd/ufe/UsdUndoDuplicateCommand.h

```diff
@@ -42,6 +42,10 @@ class MAYAUSD_CORE_PUBLIC UsdUndoDuplicateCommand : public Ufe::UndoableCommand
     typedef std::shared_ptr<UsdUndoDuplicateCommand> Ptr;
 
     UsdUndoDuplicateCommand(const UsdSceneItem::Ptr& srcItem);
+    UsdUndoDuplicateCommand(
+        const UsdSceneItem::Ptr& srcItem,
+        const UsdSceneItem::Ptr& dstParentItem);
```

Comment:
I've modified this existing so we can have a different parent item as destination and so also having a different destination layer.